docs/model-monitoring/index.md

```diff
@@ -139,6 +139,9 @@ When ramping up the scale of your model monitoring, take note of these limitatio
 - Up to 20 large projects (model endpoints per project between 1k and 5k)
 - Up to 100 medium projects (100 < model endpoints < 1k)
 - Up to 200 small projects (model endpoints < 100)
+- Up to 50,000 app results per project (V3IO). This can vary depending on the overall general load on the system and the number of nodes.
+- Up to 50,000 app results per project (TSDB). This can vary depending on the overall general load on the system and the number of nodes.
+- The maximum number of projects depends on the scalability of the storage system. In the case of TSDB, it also depends on the TSDB capacity for handling requests
```

Comment:
On each project and per 10 minute `base-period`:
- up to 50K results/metrics can be captured (V3IO-TSDB)
- up to 5K results/metrics can be captured (Tdengine-TSDB)
These numbers can vary depending on the overall system "stress" level and the TSDB performance. 
An example of suitable V3io-TSDB setup: A project with model-monitoring enabled with 10 minute base-period, with 5 serving functions each with 100 models, and two model-monitoring-apps each with 5 results: 5 serving-functions * 1000 models * 2 mm-apps * 5 results = 50000 results / 10 min base-period

Reply:
updated